mod/photos.php

```diff
@@ -1411,17 +1411,15 @@ function photos_content(App $a)
 		if (count($linked_items) && strlen($link_item['tag'])) {
 			$arr = explode(',', $link_item['tag']);
 			// parse tags and add links
-			$tag_str = '';
-			foreach ($arr as $t) {
-				if (strlen($tag_str)) {
-					$tag_str .= ', ';
-				}
-				$tag_str .= BBCode::convert($t);
+			$tag_arr = [];
+			foreach ($arr as $tag) {
+				array_push($tag_arr, ['name' => BBCode::convert($tag), 
+					'removeurl' => '/tagrm/'.$link_item['id'] . '/' . bin2hex($tag)]);
 			}
-			$tags = [L10n::t('Tags: '), $tag_str];
+			$tags = ['title' => L10n::t('Tags: '), 'tags' => $tag_arr];
 			if ($cmd === 'edit') {
-				$tags[] = 'tagrm/' . $link_item['id'];
-				$tags[] = L10n::t('[Remove any tag]');
+				$tags += ['removeanyurl' => 'tagrm/' . $link_item['id']];
```

Comment:
Is this another form of ```$tags[] = ```?

Reply:
More like `$tags['removeanyurl'] = 'tagrm/' . $link_item['id']`;

Reply:
It the concatenation operation. You can merge two arrays like the `$str .= 'abc'` see here https://friendica.riotcat.org/display/91bfdaa4-195b-ce0d-a0ec-86a397945817